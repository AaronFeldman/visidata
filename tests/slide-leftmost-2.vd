sheet	col	row	longname	input	keystrokes	comment
	SqliteSheet	header	set-option	0		
	UsvSheet	delimiter	set-option	␞		
	UsvSheet	row_delimiter	set-option	␟		
			open-file	sample_data/benchmark.csv	o	
benchmark	Date		key-col		!	
benchmark	Item		slide-leftmost		gH	
