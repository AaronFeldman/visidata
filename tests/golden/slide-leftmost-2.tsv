Date	Item	Customer	SKU	Quantity	Unit	Paid
7/3/2018 1:47p	BFF Oh My Gravy! Beef & Salmon 2.8oz	Robert Armstrong	FOOD213	4	$12.95	$51.8
7/3/2018 3:32p	Food, Adult Cat - 3.5 oz	Kyle Kennedy	FOOD121	1	$4.22	$4.22
7/5/2018 4:15p	Food, Adult Cat 3.5 oz	Douglas "Dougie" Powers	FOOD121	1	$4.22	$4.22
7/6/2018 12:15p	Food, Senior Wet Cat - 3 oz	桜 高橋 (Sakura Takahashi)	FOOD122	12	$1.29	157¥
7/10/2018 10:28a	Food, Salamander	David Attenborough	NSCT201	30	$.05	$1.5
7/10/2018 5:23p	Cat, Korat (Felis catus)	Susan Ashworth	CAT060	1	$720.42	$720.42
7/10/2018 5:23p	Food, Kitten 3kg	Susan Ashworth	FOOD130	1	$14.94	$14.94
7/13/2018 10:26a	Monster, Rust (Monstrus gygaxus)	Wil Wheaton	NSCT523	1	$39.95	$39.95
7/13/2018 3:49p	BFF Oh My Gravy! Chicken & Shrimp 2.8oz	Robert Armstrong	FOOD216	4	$12.95	$51.8
7/17/2018 9:01a	BFF Oh My Gravy! Duck & Tuna 2.8oz	Robert Armstrong	FOOD217	4	$12.95	$51.8
7/17/2018 11:30a	Rabbit (Oryctolagus cuniculus)	Helen Halestorm	LAGO342	2	$32.94	$65.88
7/18/2018 12:16p	Food, Senior Wet Cat - 3 oz	桜 高橋 (Sakura Takahashi)	FOOD122	6	$1.29	157¥
7/19/2018 10:28a	Food, Dog - 5kg	Rubeus Hagrid	FOOD170	5	$44.95	$224.75
7/20/2018 2:13p	Food, Premium Wet Cat - 3.5 oz	Jon Arbuckle	FOOD167	50	$3.95	$197.5
7/23/2018 1:41p	BFF Oh My Gravy! Lamb & Tuna 2.8oz	Robert Armstrong	FOOD215	4	$12.95	$51.8
7/23/2018 4:23p	Laser Pointer	Douglas "Dougie" Powers	TOY235	1	$16.12	$16.12
7/24/2018 12:16p	Food, Senior Wet Cat - 3 oz	桜 高橋 (Sakura Takahashi)	FOOD122	3	$1.29	157¥
7/26/2018 4:39p	Food, Shark - 10 kg	Douglas "Dougie" Powers	FOOD420	1	$15.70	$15.7
7/27/2018 12:16p	Food, Senior Wet Cat - 3 oz	桜 高橋 (Sakura Takahashi)	FOOD122	3	$1.29	157¥
7/30/2018 12:17p	Food, Senior Wet Cat - 3 oz	桜 高橋 (Sakura Takahashi)	RETURN	1	$1.29	157¥
7/31/2018 5:42p	Food, Dragon - 50kg	Rubeus Hagrid	CAT060	5	$720.42	$3602.1
8/1/2018 2:44p	Food, Rhinocerous - 50kg	David Attenborough	FOOD360	4	$5.72	$22.88
8/2/2018 5:12p	Cat, Maine Coon (Felix catus)	Susan Ashworth	CAT110	1	$1,309.68	$1309.68
8/2/2018 5:12p	Food, Kitten 3kg	Susan Ashworth	FOOD130	3	$14.94	$44.82
8/6/2018 10:21a	BFF Oh My Gravy! Beef & Chicken 2.8oz	Robert Armstrong	FOOD212	4	$12.95	$51.8
8/7/2018 4:12p	Kingsnake, California (Lampropeltis getula)	Juan Johnson	REPT082	1	$89.95	$89.95
8/7/2018 4:12p	Mouse, Pinky (Mus musculus)	Juan Johnson	RDNT443	1	$1.49	$1.49
8/10/2018 4:31p	BFF Oh My Gravy! Chicken & Turkey 2.8oz	Robert Armstrong	FOOD211	4	$12.95	$51.8
8/13/2018 2:07p	Mouse, Pinky (Mus musculus)	Monica Johnson	RDNT443	1	$1.49	$1.49
8/13/2018 2:08p	Forti Diet Prohealth Mouse/Rat 3lbs	María Fernández	FOOD146	2	$2.00	$4.0
8/15/2018 11:57a	Parrot, Norwegian Blue (Mopsitta tanta)	Mr. Praline	RETURN	1	$2300.00	-$2300.0
8/15/2018 3:48p	Food, Adult Cat - 3.5 oz	Kyle Kennedy	FOOD121	2	$4.22	$8.44
8/16/2018 11:50a	Rabbit (Oryctolagus cuniculus)	Helen Halestorm	RETURN	6	$0	$0.0
8/16/2018 4:00p	Dog, Golden Retriever (Canis lupus familiaris)	Kyle Kennedy	DOG010	1	$2,495.99	$2495.99
8/16/2018 5:15p	Parakeet, Blue (Melopsittacus undulatus)	Michael Smith	BIRD160	1	29.95	$31.85
8/17/2018 9:26a	Food, Spider	Rubeus Hagrid	NSCT201	5	$.05	$0.25
8/20/2018 9:36a	Dog, Golden Retriever (Canis lupus familiaris)	Kyle Kennedy	RETURN	1	$1,247.99	-$1247.99
8/20/2018 1:47p	Goat, American Pygmy (Capra hircus)	מרוסיה ניסנהולץ אבולעפיה	GOAT224	1	₪499	$160.51
8/20/2018 3:31p	Crickets, Adult Live (Gryllus assimilis)	Monica Johnson	NSCT201	30	$.05	$1.5
8/20/2018 5:12p	Food, Pangolin	David Attenborough	NSCT084	30	$.17	$5.10
8/21/2018 12:13p	BFF Oh My Gravy! Duck & Salmon 2.8oz	Robert Armstrong	FOOD214	4	$12.95	$51.8
8/22/2018 9:38a	Food, Quoll	David Attenborough	BIRD160	1	29.95	$29.95
8/22/2018 2:13p	Food, Adult Dog - 5kg	Jon Arbuckle	FOOD170	1	$44.95	$44.95
8/22/2018 5:49p	Fire Extinguisher, kitchen-rated	מרוסיה ניסנהולץ	SFTY052	1	$61.70	$61.70
8/24/2018 11:42a	BFF Oh My Gravy! Chicken & Salmon 2.8oz	Robert Armstrong	FOOD218	4	$12.95	$51.8
8/27/2018 3:05p	Mealworms, Large (Tenebrio molitor) 100ct	Monica Johnson	NSCT443	1	$1.99	$1.99
8/28/2018 5:32p	Cat, Scottish Fold (Felis catus)	Susan Ashworth	CAT020	1	$1,964.53	$1964.53
8/28/2018 5:32p	Food, Kitten 3kg	Susan Ashworth	FOOD130	2	$14.94	$29.88
8/29/2018 10:07a	BFF Oh My Gravy! Chicken & Pumpkin 2.8oz	Robert Armstrong	FOOD219	4	$12.95	$51.8
8/31/2018 12:00a	BFF Oh My Gravy! Chicken & Pumpkin 2.8oz	Robert Armstrong	FOOD219	144	$12.95	$1864.8
8/31/2018 5:57p	Lizard, Spinytail (Uromastyx ornatus)	Juan Johnson	REPT217	1	$99.95	$99.95
